Client/Frontend/Browser/Tabs/TabTrayV2ViewController.swift

```diff
@@ -135,10 +164,21 @@ extension TabTrayV2ViewController: UITableViewDataSource {
                                completion()
                            })
        }
-    
+
     @objc func dismissTabTray() {
+        // We check if there is private tab then add one if user dismisses
+        viewModel.addPrivateTab()
         navigationController?.dismiss(animated: true, completion: nil)
     }
+    
+    @objc func didTapLearnMore() {
+        let appVersion = Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String
+        if let langID = Locale.preferredLanguages.first {
+            let learnMoreRequest = URLRequest(url: "https://support.mozilla.org/1/mobile/\(appVersion ?? "0.0")/iOS/\(langID)/private-browsing-ios".asURL!)
```

Comment:
There is also `SupportUtils.URLForTopic()` that abstracts away the above three lines of code to avoid some duplication.

Reply:
Thanks, that cleaned up the method really well :)